curvefs/proto/metaserver.proto

```diff
@@ -330,6 +334,63 @@ message GetOrModifyS3ChunkInfoResponse {
     map<uint64, S3ChunkInfoList> s3ChunkInfoMap = 3;
 }
 
+message InodeAttr {
```

Comment:
> There is a lot of repetition with inode here, can it be abstracted?

Yes, can change Inode to like 
{
 attr
xattr
volumeExtentList
s3ChunkInfoMap
};

But i tried this form at first and it will change all the files exists. If need, can fix in another pr later.